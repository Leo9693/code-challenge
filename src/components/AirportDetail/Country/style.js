import styled, { keyframes } from "styled-components";
import { fadeInRight } from "react-animations";

export const AnimatedCountry = styled.div`
    animation: ${keyframes`${fadeInRight}`} 1.6s;
`;
