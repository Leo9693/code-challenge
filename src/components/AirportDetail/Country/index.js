import React from "react";
import { AnimatedP } from "../style";
import { AnimatedCountry } from "./style";
function index(props) {
    const { countryCode, countryName } = props.country;
    return (
        <AnimatedCountry>
            <h3>Country:</h3>
            <AnimatedP>CountryCode: {countryCode}</AnimatedP>
            <AnimatedP>CountryName: {countryName}</AnimatedP>
        </AnimatedCountry>
    );
}

export default index;
