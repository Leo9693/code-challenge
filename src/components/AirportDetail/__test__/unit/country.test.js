import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Country from "../../Country";
Enzyme.configure({ adapter: new Adapter() });

test("test country component", () => {
    const wrapper = shallow(
        <Country
            country={{
                countryCode: "",
                countryName: ""
            }}
        />
    );
    expect(wrapper.debug()).toMatchSnapshot();
});
