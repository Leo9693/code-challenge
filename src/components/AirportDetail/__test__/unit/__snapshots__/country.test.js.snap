// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test country component 1`] = `
"<styled.div>
  <h3>
    Country:
  </h3>
  <styled.p>
    CountryCode: 
  </styled.p>
  <styled.p>
    CountryName: 
  </styled.p>
</styled.div>"
`;
